featuregates/client/go.mod

```diff
@@ -11,6 +11,7 @@ replace (
 	github.com/vmware-tanzu/tanzu-framework/cli/core => ../../cli/core
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../cli/runtime
 	github.com/vmware-tanzu/tanzu-framework/packageclients => ../../packageclients
+	github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkr => ../../pkg/v1/tkr
```

Comment:
Is this change necessary?

Reply:
this is an unrelated change, but the featuregates client module was complaining about this.. since it is a small change, I thought I will push it with this change.